trade-finance-manager-ui/templates/case/amendments/amendment-answers.njk

```diff
@@ -80,7 +80,7 @@
     %}
     {{- optionsWithHTML.push(defaultCoverEndDate) | reject() }}
 
-    {% if isTfmFacilityEndDateFeatureFlagEnabled %}
+    {% if showFacilityEndDate %}
```

Comment:
is this called `shouldDisplayFacilityEndDate` in the facility_details macro - are they intentionally different

Reply:
Good spot, I've changed it everywhere to `showFacilityEndDate` for consistency